token-client/src/main/java/com/sap/cloud/security/xsuaa/client/OAuth2TokenServiceConstants.java

```diff
@@ -0,0 +1,20 @@
+package com.sap.cloud.security.xsuaa.client;
+
+public class OAuth2TokenServiceConstants {
```

Comment:
you may add here also the bearer types, saml and jwt for future use....

Reply:
I will add jwt, as soon i can support the endpoints (xsuaa needs to provide a fix first)